src/views/admin-consent/ScopeList.js

```diff
@@ -22,6 +22,7 @@ const SCOPE_N_GROUP_CONFIG = {
   authenticators: 'resource',
   authorizationServers: 'resource',
   clients: 'resource',
+  domains: 'resource',
```

Comment:
add a unit test for this

Reply:
marking it resolved as discussed with @pradeepdewda-okta